### PR TITLE
Fixed attribute error-- on changing the permission for a person without email"

### PR DESCRIPTION
**Pull request by prerit2010**

### CODE REVIEW

File: workshops/models.py [hunk 1]
```diff
@@ -332,7 +332,10 @@ def clean(self):
         """This will be called by the ModelForm.is_valid(). No saving to the
         database."""
         # lowercase the email
-        self.email = self.email.lower()
```
Comment (narayanaditya95) [inline workshops/models.py]: How about `self.email = self.email.lower() if self.email`.

Comment (prerit2010) [inline workshops/models.py]: @narayanaditya95  Then it should be : `self.email = self.email.lower() if self.email else None`

**Comment by pbanaszkiewicz** [comment]
Looks good. @prerit2010 can you add a regression test to `test_person.TestPerson`? Thanks a lot.

**Comment by pbanaszkiewicz** [comment]
@prerit2010 thanks.

Ref #840.

**Comment by pbanaszkiewicz** [comment]
Since we have #844 scheduled for v1.6, let's fix this bug in this release as well. Thanks @prerit2010 for this contribution!

**Pull request merged**